index.html

```diff
@@ -33,18 +38,192 @@ <h2> I'am a software engineer and mecatronic engineer</h2>
                 contact me.</em><br></p>
         </div>
 <div>
-    <ul>
+    <ol>
         <li>
-          <a href="https://github.com/cvalencia1991"><img src="imgs/githubimg.png" alt="githubimage"></a>
+          <a href="#githublink" ><img src="imgs/githubimg.png" alt="githubimage"></a>
         </li>
         <li>
-            <a href="https://www.linkedin.com/in/cvalenciaguilar/"><img  src="imgs/linkeinimg.png" alt="linkedinimage"></a>
+            <a href="#linkeinlink" ><img  src="imgs/linkeinimg.png" alt="linkedinimage"></a>
         </li>
         <li>
-            <a href="https://twitter.com/cvalenciaguilar"><img src="imgs/twitter.png" alt="twitterimg"></a>
+            <a href="#twitterinlink" ><img src="imgs/twitter.png" alt="twitterimg"></a>
         </li>
+    </ol>
```

Comment:
### OPTIONAL

- [ ] Kindly use a ul tag to wrap the icons instead of an ol tag 👍🏿 
- [ ]  kindly add the angel and medium icons from the design, please add it 👍🏿

Reply:
ok i edit it